src/openms/source/ANALYSIS/ID/MetaboliteSpectralMatching.cpp

```diff
@@ -404,7 +404,7 @@ namespace OpenMS
       double db_intensity = 0.0;
       for (const auto& db_it : match.second)
       {
-        db_intensity = max(db_intensity, double(db_it->getIntensity()));
+        db_intensity = max(db_intensity, double(db_it->getIntensity())); //FIXME should this be the sum of all the intensities / number of intensities?
```

Comment:
Maybe it gets normalized earlier?

Reply:
Not that I could find. The formal definition of Hyperscore just counts them once (https://www.inf.fu-berlin.de/lehre/WS14/ProteomicsWS14/LUS/lu7b/427/index.html), which isn't especially surprising as this case comes up way less commonly in peptides. I can change this to a less action oriented NOTE in the code.

Reply:
ah I think this is correct. if a spectrum peak matches several db peaks it will take the highest intensity one in the spectra db as reference.